concept/type_/owns.rs

```diff
@@ -200,36 +237,59 @@ impl<'a> InterfaceImplementation<'a> for Owns<'a> {
     fn interface(&self) -> AttributeType<'a> {
         self.attribute.clone()
     }
-
-    fn unwrap_annotation(annotation: OwnsAnnotation) -> Annotation {
-        match annotation {
-            OwnsAnnotation::Distinct(distinct) => Annotation::Distinct(distinct),
-            OwnsAnnotation::Key(key) => Annotation::Key(key),
-            OwnsAnnotation::Cardinality(cardinality) => Annotation::Cardinality(cardinality),
-            OwnsAnnotation::Unique(unique) => Annotation::Unique(unique),
-        }
-    }
 }
 
-#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
 pub enum OwnsAnnotation {
     Distinct(AnnotationDistinct),
     Unique(AnnotationUnique),
     Key(AnnotationKey),
     Cardinality(AnnotationCardinality),
+    Regex(AnnotationRegex),
+}
+
+impl OwnsAnnotation {
```

Comment:
I'll name it `DefaultAnnotationFromCategory` as we want to return `AnnotationError` instead of `ConversionError`.
If we need another `Default***FromCategory` in the future, we could make `ConversionError` containing `source: AnnotationError` or `***Error` for something else, renaming the trait to `DefaultFromCategory`. It is redundant now, I guess

Reply:
Done! It looks a bit more verbose but does the job =)